src/redisearch_rs/inverted_index/src/index_result.rs

```diff
@@ -665,7 +665,7 @@ impl<'index> RSIndexResult<'index> {
             doc_id: 0,
             dmd: ptr::null(),
             field_mask: 0,
-            freq: 1,
+            freq: 0,
```

Comment:
I don't remember exactly, but didn't we agreed that `freq 1` was a better default?
@GuyAv46 ?

Reply:
It broke the tests (this PR is finally activating this code). That's why I changed it back to align with the current behaviour

https://github.com/RediSearch/RediSearch/blob/fa08b0bb63d04b74284b0014f6f16e8e91d93036/src/index_result.c#L19-L34